secure/storage/aws/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "libra-secure-storage-aws"
+version = "0.1.0"
+authors = ["Libra Association <opensource@libra.org>"]
+repository = "https://github.com/libra/libra"
+description = "Libra's Persistent, Secure Storage implemted using AWS SecretsManager"
+homepage = "https://libra.org"
+license = "Apache-2.0"
+publish = false
+edition = "2018"
+
+[dependencies]
+rusoto_secretsmanager = {version = "0.42.0", default-features = false, features=["rustls"]}
```

Comment:
Would recommend looking at how we implemented our Vault client. Ideally if we have all the same dependencies, we can compile a single version of our various TCBs for these different environments. Otherwise in order to minimize dependencies, we'll need multiple binaries, which doesn't seem very fun from an operational perspective.

Reply:
yeah, I will first use rusoto to get something working, then switch to use ureq